### DISCUSSION TITLE
Wish List

### DISCUSSION THREAD

**Discussion by sfranzyshen** [cesanta/elk#38] [Ideas]
Support for additional ES6 functionality:

**Arrays:**
```
let fruits = ['Apple', 'Banana']

console.log(fruits.length)
// 2
```
**For() Loops:**
```
for (i = 0; i < 5; i++) {
  text += "The number is " + i + "<br>";
}
```

**Comment by cpq** [comment]
That is unlikely to be implemented - see https://github.com/cesanta/elk/discussions/37

For any sort of feature request, we require a real life use case: why an existing feature set is not good. For both examples: 1 - objects can be used, 2 - while loop can be used: `let i = 0; while (i++ < 10) true;`
